.travis.yml

```
@@ -198,10 +198,12 @@ jobs:
 
     - stage: Publish Docker Registry
       name: Publish Docker Registry
-      script: ./scripts/travis/deploy/publish.sh "app"
+      script: ./scripts/travis/deploy/publish.sh "app" "$DOCKER_REPOSITORY_DOMAIN" "$QUAY_USERNAME" "$QUAY_PASSORD"
 
-    - stage: Release Tag
-      script: ./scripts/travis/release/git-tag.sh
+    - stage: Release Tag and Publish to Dockerhub
+      script:
+        - ./scripts/travis/release/git-tag.sh
+        - ./scripts/travis/deploy/publish.sh "app" "$DOCKER_HUB_REPOSITORY_DOMAIN" "$DOCKER_HUB_USERNAME" "$DOCKER_HUB_PASSWORD"
```

better to use a different job

```suggestion
        - stage: Release Tag and Publish to Dockerhub
      name: Release Tag
      script: ./scripts/travis/release/git-tag.sh
    
    - stage: Release Tag and Publish to Dockerhub
      name: Publish to Dockerhub
      script: ./scripts/travis/deploy/publish.sh "app" "$DOCKER_HUB_REPOSITORY_DOMAIN" "$DOCKER_HUB_USERNAME" "$DOCKER_HUB_PASSWORD"
```

---

Done - slightly differently - avoid repeating the stage: